src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -49,25 +49,15 @@ public class LogHelper
         /// </summary>
         private static bool _isHeaderWritten { get; set; } = false;
 
-        /// <summary>
-        /// The log message that indicates the current library version.
-        /// </summary>
-        private static string _versionLogMessage = "Microsoft.IdentityModel version: {0}. ";
-
-        /// <summary>
-        /// The log message that indicates the date.
-        /// </summary>
-        private static string _dateLogMessage = "Date: {0}. ";
-
         /// <summary>
         /// The log message that is shown when PII is off.
         /// </summary>
-        private static string _piiOffLogMessage = "PII (personally identifiable information) logging is currently turned off. Set IdentityModelEventSource.ShowPII to 'true' to view the full details of exceptions. ";
+        private const string PiiOffLogMessage = "PII logging is off. See https://aka.ms/wilson/pii for details. ";
```

Comment:
off and on could be displayed in uppercase (OFF/ON) to highlight the value. Looks like we have a PII link we can use here - https://aka.ms/IdentityModel/PII